### PR TITLE
Observability draft

### PR DESCRIPTION
**Pull request by dockerymick**
* https://github.com/open-cluster-management/backlog/issues/5141

* https://github.com/open-cluster-management/backlog/issues/4775

* https://github.com/open-cluster-management/backlog/issues/5235

* https://github.com/open-cluster-management/backlog/issues/4621

### CODE REVIEW

**Comment by dockerymick** [comment]
@mqhackett I mentioned that the must-gather command can be used for Observability on the new file that i created ../console/observability.adoc

File: console/console.adoc [hunk 1]
```diff
@@ -1,11 +1,8 @@
-[#observability-in-the-console]
-= Observability in the console
+[#red-hat-advanced-cluster-management-for-kubernetes-console-components]
+= {product-title} console components
```
Comment (swopebe) [inline console/console.adoc:3]: Without the product name

File: about/components.adoc [hunk 2]
```diff
@@ -40,7 +40,7 @@ This is a service that should not be accessed directly.
 
 [#default-enabled-hub-cluster-components]
 == Default enabled hub cluster components
-
+// do we need to add multicluster-obeservability-operator? Currently, i doc'ed that the service is disabled by default
```
Comment (swopebe) [inline about/components.adoc:43]: makes sense

File: console/observability.adoc [hunk 3]
```diff
@@ -0,0 +1,44 @@
+[#install-observability]
+= Install observability service
+
```
Comment (swopebe) [inline console/observability.adoc]: Would be your first topic in the parent topic.

File: release_notes/whats_new.adoc [hunk 4]
```diff
@@ -24,6 +24,8 @@ See link:../install/install_connected.adoc#installing-while-connected-online[Ins
 You can now provision and manage {ocp} clusters on the VMware vSphere Kubernetes cloud service provider.
 See link:../manage_cluster/create_vm.adoc#creating-a-cluster-on-vmware-vsphere[Creating a cluster on VMware vSphere] for more information. 
 
+* Install the observability service (`multicluster-observability-operator`) to monitor and optimize the health of your managed clusters. You can view the collected data and logs from Grafana dashboards. For more information see, _Observability service_ section in the link:../console/observability.adoc[{product-title} console components documentation].
```
Comment (swopebe) [inline release_notes/whats_new.adoc]: You can now...

Change link when you get done with restructure.

File: observability/master.adoc [hunk 5]
```diff
@@ -0,0 +1,6 @@
+include::modules/common-attributes.adoc[]
+
+= Observing environments
+
+include::observe_intro.adoc[leveloffset=+1]
+include::observability_install.adoc[leveloffset=+2]
```
Comment (swopebe) [inline observability/master.adoc:6]: looks great

**Review by swopebe — APPROVED** [review]

File: about/components.adoc (hunk 2 above)
Comment (dockerymick) [reply about/components.adoc:43]: I will keep my comment here bc the observability service is not enabled by default.

**Comment by dockerymick** [comment]
I will continue to merge this PR. I made a note on the issue here: https://github.com/open-cluster-management/backlog/issues/4911#issuecomment-697766744

**Pull request merged**